### PR TITLE
exercicio 2

### PR DESCRIPTION
**Pull request by thiagopinheirolima**

### CODE REVIEW

File: Exercicio 2/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+            this.InitializeComponent();
+        }
+
```
Comment (soutolucas) [inline Exercicio 2/MainPage.xaml.cs]: Could you please rename methods to a more suggestive name?

File: Exercicio 2/Page1.xaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    x:Class="Exercicio_2.Page1"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Exercicio_2"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Background="Red">
+        <StackPanel VerticalAlignment="Center" HorizontalAlignment="Center">
```
Comment (soutolucas) [inline Exercicio 2/Page1.xaml]: Is it really necessary to use "StackPanel" here?

File: Exercicio 2/Page1.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    /// </summary>
+    public sealed partial class Page1 : Page
+    {
+        public Page1()
+        {
+            this.InitializeComponent();
+        }
+
+        protected override void OnNavigatedFrom(NavigationEventArgs e)
+        {
```
Comment (soutolucas) [inline Exercicio 2/Page1.xaml.cs]: Please consider move it to App,OnLauched.

File: Exercicio 2/Page1.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            SystemNavigationManager.GetForCurrentView().AppViewBackButtonVisibility = AppViewBackButtonVisibility.Collapsed;
+            SystemNavigationManager.GetForCurrentView().BackRequested -= OnBackResquested;
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            SystemNavigationManager.GetForCurrentView().AppViewBackButtonVisibility = AppViewBackButtonVisibility.Visible;
+            SystemNavigationManager.GetForCurrentView().BackRequested += OnBackResquested;
```
Comment (soutolucas) [inline Exercicio 2/Page1.xaml.cs]: Please consider move the above code to App.xaml.cs. This code can be reuse on others pages.

File: Exercicio 2/Page2.xaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    x:Class="Exercicio_2.Page2"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Exercicio_2"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Background="Blue">
+        <StackPanel VerticalAlignment="Center" HorizontalAlignment="Center">
```
Comment (soutolucas) [inline Exercicio 2/Page2.xaml]: Please, consider remove "StackPanel".

File: Exercicio 2/Page1.xaml.cs (hunk 3 above)
Comment (thiagopinheirolima) [reply Exercicio 2/Page1.xaml.cs]: I know that I could re-use code here, but the training video explains that here's the right place to put that code, so I'm just following the instructions.

File: Exercicio 2/Page1.xaml.cs (hunk 4 above)
Comment (thiagopinheirolima) [reply Exercicio 2/Page1.xaml.cs]: Duplicated

**Pull request merged**